pgrx/src/fcinfo.rs

```diff
@@ -80,6 +80,7 @@ macro_rules! name {
     };
 }
 
+#[deprecated(since = "0.12.0", note = "try VariadicArray")]
```

Comment:
Just delete the macro.  There's enough breaking changes in develop that one more doesn't matter.

Reply:
All the real parts of the macro, and thus the things that complicate its support internally, are gone. There are other users that don't have their code published, and for them, telling them what else to do may make migration slightly easier.

Reply:
Obviously I've broken probably a fair number of unpublished extensions with this version but it seems low-effort.

Reply:
can it become a compile error instead of a deprecated warning?  Cuz using it now would mean it doesn't dooooooooo what they want, right?

Reply:
oh sure.

Reply:
actually it already is lol